numba_dpex/tests/dpjit_tests/dpnp/test_dpnp_empty.py

```diff
@@ -139,3 +139,22 @@ def func(shape, queue):
     with pytest.raises(errors.TypingError):
         queue = dpctl.SyclQueue()
         func(10, queue)
+
+
+def test_dpnp_empty_with_dpjit_queue():
+    """Tests basic boxing and unboxing of a dpnp.ndarray object.
+
+    Checks if we can pass in and return a dpctl.ndarray object to and
+    from a dpjit decorated function.
+    """
+
+    @dpjit
+    def func(a):
+        queue = a.sycl_queue
+        return dpnp.empty(10, sycl_queue=queue)
+
+    a = dpnp.empty(10)
+    b = func(a)
+
+    assert b is not None
+    assert b.sycl_queue is not None
```

Comment:
we should also be able to assert `b.sycl_queue == a.sycl_queue`?

Reply:
It will fail then. I don't know how to add just one assert as expected failure

Reply:
Why will it fail? The `__eq__` of the `dpctl.SyclQueue` class just calls the `DPCTLQueue_Equal` function that should evaluate to true if the `qref` pointers point to the same underlying `sycl::queue` object.

Reply:
We must be more specific. It suppose to be same object. Marked test as XFAIL.